### ISSUE TITLE
Question about the performance on seg-face

### ISSUE THREAD

**Issue by gzhhhere** [eladrich/pixel2style2pixel#243]
Hi! i tested your pretrained model "psp_celebs_seg_to_face.pt" on celebea_Mask dataset, however, i got bad performance on hair.
![1](https://user-images.githubusercontent.com/43848061/149625695-0e1f7be3-b323-412f-a773-1153f77f1efd.png)

i used https://github.com/zllrunning/face-parsing.PyTorch/blob/master/prepropess_data.py to prepropess my mask-anno. What could be the reason for the bad performance? i noticed my mask color in inference_coupled is different from yours in PSP paper.

**Comment by gzhhhere** [comment]
i think the reason could be we use different number for the attributes.  if this is true, how can i get the same performance as yours, can you give me the list you used in your mask-anno prepropessing?

**Comment by gzhhhere** [comment]
> 

    atts = ['skin', 'l_brow', 'r_brow', 'l_eye', 'r_eye', 'eye_g', 'l_ear', 'r_ear', 'ear_r',
            'nose', 'mouth', 'u_lip', 'l_lip', 'neck', 'neck_l', 'cloth', 'hair', 'hat'] 
that is, whats your 'atts list'... ... Thank you in advance for your kind help.

**Comment by gzhhhere** [comment]
> 

or you can easily give me one example of your input mask... ...

**Comment by yuval-alaluf** [comment]
There are two reasons for the difference: 
1. The attributes are different as you said. Our data was taken from the official CelebA-Mask dataset (https://github.com/switchablenorms/CelebAMask-HQ/tree/master/face_parsing). There you can find the order of the labels which differ from your labels. 
2. During inference we use style mixing to get the final results. Details on how the style mixing is performed is provided in the paper and we have provided a script for reproducing the results in the paper. 

> or you can easily give me one example of your input mask... ...
The link above has all the information you need to generate the data yourself include how to download the data and the order of the annotations.

**Comment by gzhhhere** [comment]
> There are two reasons for the difference:
> 
> 1. The attributes are different as you said. Our data was taken from the official CelebA-Mask dataset (https://github.com/switchablenorms/CelebAMask-HQ/tree/master/face_parsing). There you can find the order of the labels which differ from your labels.
> 2. During inference we use style mixing to get the final results. Details on how the style mixing is performed is provided in the paper and we have provided a script for reproducing the results in the paper.
> 
> > or you can easily give me one example of your input mask... ...
> > The link above has all the information you need to generate the data yourself include how to download the data and the order of the annotations.

Thank your for your kind help!
